jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/sender/PushNotificationSenderEndpoint.java

```diff
@@ -99,12 +102,21 @@
      *
      * @HTTP 202 (Accepted) Indicates the Job has been accepted and is being process by the AeroGear UnifiedPush Server.
      * @HTTP 401 (Unauthorized) The request requires authentication.
-     * @HTTP 404 (Not Found) The requested PushApplication resource does not exist.
      * @RequestHeader aerogear-sender The header to identify the used client. If the header is not present, the standard "user-agent" header is used.
+     *
+     * @param message   message to send
+     * @return          empty JSON body
+     *
+     * @responseheader WWW-Authenticate Basic realm="AeroGear UnifiedPush Server" (only for 401 response)
+     *
+     * @statuscode 202 Indicates the Job has been accepted and is being process by the AeroGear UnifiedPush Server
+     * @statuscode 401 The request requires authentication
      */
     @POST
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
+    @BodyType("org.jboss.aerogear.unifiedpush.message.UnifiedPushMessage")
```

Comment:
Looking at http://condor.pp.ua/aerogear-ups-rest-api-doc/latest/#397083935

I really like the info about the JSON type of the passed in `UnifiedPushMessage`.

Now I wonder, if we on the actual JavaDoc should avoid long example, and just give a little bit, like:
```
{
"message": {
      "alert": "HELLO!",
      "sound": "default",
      "user-data": {
          "key": "value",
      }
}
```

and reference to the full details to the JSON version of the `UnifiedPushMessage`?

@sebastienblanc @lfryc any thoughts?